datashader/core.py

```diff
@@ -246,8 +260,9 @@ def trimesh(self, vertices, simplices, mesh=None, agg=None, interp=True, interpo
             elif interpolate == 'nearest':
                 interp = False
             else:
-                raise ValueError('Invalid interpolate method: options include {}'.format(['linear','nearest']))
-            
+                raise ValueError('Invalid interpolate method: options include {}'
+                                 .format(['linear', 'nearest']))
+
```

Comment:
It would be very helpful to revert whitespace and source-code formatting changes like these (here and in other cases below), because they make it much more difficult to see the substantive changes being proposed.  It's fine to change whitespace if it's not well formatted, but it should be in a separate PR that we can verify isn't changing anything.

Reply:
Reverting formatting changes for the purpose of this PR.